### PR TITLE
DOCS-15434 read write concern v0.9.0

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCS-15434-read-write-concern-v0.9.0/faq/#which-connection-string-options-does-mongosync-allow-)

[JIRA](https://jira.mongodb.org/browse/DOCS-15434)

### CODE REVIEW

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM

File: source/faq.txt [hunk 1]
```diff
@@ -56,9 +56,13 @@ Which connection string options does ``mongosync`` allow?
...
-and :ref:`writeConcern: "majority" <write-concern>`. ``mongosync`` 
-overwrites any other ``readConcern`` or ``writeConcern`` and uses
-``"majority"`` instead.
+and :ref:`writeConcern: "majority" <write-concern>`. If the
+``readConcern`` or ``writeConcern`` is not ``majority``, ``mongosync`` 
+returns an error:
+
+.. code-block:: shell
+
+   Invalid URI option, write/read concern must be majority
```
Comment (ali-mir) [inline source/faq.txt:65]: My apologies, this is my miscommunication: if the invalid options is `write concern`, then the error will be:

```
Invalid URI option, write concern must be majority
```

if the invalid option is `read concern`, the error will be

```
Invalid URI option, read concern must be majority
```

The current state of the documentation looks good to me, but I'm not sure if the clarification would change how you phrase it.

Comment (ali-mir) [reply source/faq.txt:65]: If you still want to go ahead with this change, let me know and I can approve it. Sorry for the confusion again!

Comment (davemungo) [reply source/faq.txt]: No worries, I've updated the text

**Review by davemungo — COMMENTED** [review]
@ali-mir thanks for your feedback, this is ready for another pass

**Review by ali-mir — APPROVED** [review]
this looks great!

**Pull request merged**